deployment/terraform/ssh/ssh.go

```diff
@@ -56,7 +58,7 @@ func (sshc *Client) RunCommand(cmd string) error {
 }
 
 // Upload uploads a given src object to a given destination file.
-func (sshc *Client) Upload(src io.Reader, sudo bool, dst string) error {
+func (sshc *Client) Upload(src io.Reader, dst string, sudo bool) error {
```

Comment:
Would using `Copy` instead of `Upload` more conventional?

Reply:
mmm .. Copy can mean copying within the same machine. Upload is more clear IMO that it's uploading a file. I think I have seen both being used.

But 0/5 frankly. If you feel more than 2/5 on it, we can change.

Reply:
If I may say, I would keep `Upload`. The only reason to name it `Copy` would be if it was used by something else as an implementation of a `Copier` interface.

Reply:
It's just a thought, I was thinking for `scp` command actually.

Reply:
The thought is perfectly valid. To me this is just about keeping it a bit less abstract since the abstraction is not (yet) needed.